roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -62,25 +69,31 @@ void loadComputeLibrary() {
   }
 
 #if __GNUC__ > 5 || defined(__clang__)
-  if (__builtin_cpu_supports("avx512f")) {
-    libSuffix = "_AVX512f";
+  //skylake-avx512 support
+  if (__builtin_cpu_supports("avx512cd") && __builtin_cpu_supports("avx512vl") && __builtin_cpu_supports("avx512bw") && __builtin_cpu_supports("avx512dq"))  {
+    libSuffix = "_AVX512";
   }
 #endif
 
-#endif
 
   for (auto&& libName : libNames) {
     libName += libSuffix;
     const auto returnValue = gSystem->Load(libName.c_str());
 
     if (returnValue == -1 || returnValue == -2) {
-      throw std::runtime_error("RooFit was unable to load its computation library " + libName);
+      std::cout << "RooFit was unable to load its computation library " + libName << std::endl;
```

Comment:
Maybe add
```
if (RooFitCompute::dispatch) {
   std::cerr << "Whoops, dispatch is set even though library loading has failed?!\n";
   RooFitCompute::dispatch = nullptr;
   return;
}
```

Reply:
RooFitCompute::dispatch is actually set to `libRooFit.so`'s generic compute implementation at the beginning. If the loading fails, it should still point to that.
@manolismih  That's actually something that should be tested!